Repository/ProductRankRepository.php

```diff
@@ -23,132 +23,155 @@
 
 namespace Plugin\ProductRank\Repository;
 
-use Doctrine\ORM\EntityRepository;
 use Eccube\Entity\ProductCategory;
 use Eccube\Entity\Category;
+use Eccube\Repository\ProductCategoryRepository;
+use Doctrine\ORM\EntityManagerInterface;
 
 /**
  * ProductRank
  *
  * This class was generated by the Doctrine ORM. Add your own custom
  * repository methods below.
  */
-class ProductRankRepository extends EntityRepository
+class ProductRankRepository

```

Comment:
Please extend `AbstractRepository`.

Reply:
@lqdung-lockon @toannguyen-lockon @hoand-vn 
When extend `AbstractRepository`, I must define entity which use for repository. But ProductRank does not have any entity.